netsuite/Module.md

```diff
@@ -0,0 +1,67 @@
+## Overview
+
+NetSuite's [SuiteTalk SOAP API](https://www.netsuite.com/portal/developers/resources/suitetalk-documentation.shtml) provides capability to access NetSuite operations related different kind of NetSuite records
+such as Account, Client, Transactions, Invoice, Classifications etc.
+
+This module supports [NetSuite WSDL 2020.2.0](https://system.netsuite.com/help/helpcenter/en_US/srbrowser/Browser2020_2/schema/record/account.html) version.
+
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* A NetSuite account
+* Obtaining tokens
```

Comment:
Suggestion: "Obtain tokens."
Indent the numbered steps so that they appear correctly aligned under this bullet point.